packages/pluggableWidgets/bar-chart-native/src/BarChart.xml

```diff
@@ -37,7 +37,7 @@
                                 <caption>Data source</caption>
                                 <description>Data points which may be divided into separate series based on the "Group by" attribute value.</description>
                             </property>
-                            <property key="groupByAttribute" type="attribute" required="false" dataSource="dynamicDataSource">
+                            <property key="groupByAttribute" type="attribute" required="true" dataSource="dynamicDataSource">
```

Comment:
You can't set this to required, because it will complain for static series that it's required. You need to add a consistency check in the editor config. You can use the line chart implementation as a reference.

Reply:
Ah, thanks!

Reply:
The implementation was already there, I made a mistake after we spoke about the "required" groupByAttribute. Just the documentation needed updating (done) and not the property here.